google-cloud-spanner/lib/google/cloud/spanner/transaction.rb

```diff
@@ -1133,6 +1210,15 @@ def build_request_options options
           options
         end
 
+        ##
+        # @private Generates the next seqno in a thread-safe manner.
+        def next_seqno
```

Comment:
Maybe consider naming this `safe_next_seqno` to emphasize that we're synchronizing within this method

Reply:
Done.